Sources/Core/Microsoft.StreamProcessing/Operators/Union/PartitionedUnionPipe.cs

```diff
@@ -262,7 +258,7 @@ private void ProcessPendingEntries()
                         {
                             // Output the right tuple
                             OutputCurrentTuple(rightEntry);
-                            rightWorking.TryDequeue(out rightEntry);
+                            _ = rightWorking.TryDequeue(out _);
```

Comment:
>_ =  [](start = 28, length = 4)

why assign to a discard?

Reply:
Fixed, checking in a change momentarily.